### ISSUE TITLE
Is it possible to get rid of date/time top level directory inside exported zip files?

### ISSUE THREAD

**Issue by BengtGustafsson** [microsoft/vcpkg#13029]
I want each exported zip file to be identical for a given "vcpkg export --zip <packages>" command line. By default this is not true as a top directory including the date and time is prepended inside the zip file.

This prevents us from upgrading the used library versions without changing our cmakelists file for the using modules, which is painful.

(We have another system where you specify the name of the zip file to unpack before the build starts).

I did vcpkg help export and read the "requirement specification for export" section without finding any information in this direction. I did notice that the long list of the vcpkg help export was mostly unusable due to not specifying if a path regard _where input files come from_ or _where output files are stored_, by experimentation I found out that all options seem to relate to specifying where input files of different types are, which seems rather useless given that vcpkg decides anyway, while no option I could find controls the resulting structure of the zip file, which I guess many users may want to be able to do.

**Comment by ASxa86** [comment]
if you mean the exported `vcpkg-export-<date time>.zip` file we simply change the name of the output using the `--output` flag.

`vcpkg export zlib:x64-windows --zip --output=vcpkg-export`

**Comment by NancyLi1013** [comment]
Hi @BengtGustafsson 
Thanks for posting this issue.

Does the suggestion from @ASxa86 meet your demands?

**Comment by NancyLi1013** [comment]
@BengtGustafsson 

Please feel free to reopen this issue if it is still a problem for you.

**Comment by BengtGustafsson** [comment]
I did use --output to the name I wanted for the top directory, and then renamed the resulting zip file to a version-specific name.
I still think it has value to be able to define that the name of the top level directory in the zip file and the zip file's name are to be different.

What I did was a script which sets --output to a generic name and then renamed the output file after the month-tag checked out, to get a good traceability. After all, the checked out code version is of more interest than the date it was built.
